apps/mudita-center-e2e/src/specs/overview/e2e-mock-sample.e2e.ts

```diff
@@ -1,28 +1,35 @@
 import { E2EMockClient } from "../../../../../libs/e2e-mock/client/src"
-import {
-  outboxReloadOverview,
-  overviewDataWithoutBadge,
-} from "../../../../../libs/e2e-mock/responses/src"
+import { outboxReloadOverview } from "../../../../../libs/e2e-mock/responses/src"
+import { mockResponseTomasz } from "../../consts/mockresponse-const"
+import screenshotHelper from "../../helpers/screenshot.helper"
 
 describe("E2E mock sample - overview view", () => {
   before(async () => {
+    E2EMockClient.connect()
     //wait for a connection to be established
-    await browser.waitUntil(() => E2EMockClient.checkConnection())
+    await browser.waitUntil(() => {
+      return E2EMockClient.checkConnection()
+    })
+  })
+
+  after(() => {
+    E2EMockClient.stopServer()
+    E2EMockClient.disconnect()
   })
-  it("Connect device", async () => {
+
+  it("Test Tomasz #1 - addDevice", async () => {
```

Comment:
I guess we have similar situation like in previous file? Will this test name be change?

Reply:
yup

Reply:
Cleaned